src/components/mols/CouponCard/index.tsx

```diff
@@ -75,7 +76,7 @@ const CouponCard: React.FC<ICouponCardProps> = ({
                     />
                     {maxUnitsTotal === -1
                       ? "quantidade ilimitada"
-                      : `faltam ${maxUnitsTotal} unidades`}
+                      : getUnintsAmount(maxUnitsTotal)}

```

Comment:
- Instead of doing:
```
{unintsAmount === -1
 ? "quantidade ilimitada"
 : getUnintsAmount(unintsAmount)
}
```

- let's just do:
```
getCouponsRemaining(couponsRemaining)
```

and then handle inside the function

Reply:
done